apps/web/src/pages/templates/TemplatesListPage.tsx

```diff
@@ -143,27 +136,31 @@ function NotificationList() {
         }
       />
       <TemplateListTableWrapper>
-        <Table
-          onRowClick={onRowClick}
-          loading={isLoading || areNotificationGroupLoading}
-          data-test-id="notifications-template"
-          columns={columns}
-          data={templates || []}
-          pagination={{
-            pageSize: pageSize,
-            current: page,
-            total: totalTemplatesCount,
-            onPageChange: handleTableChange,
-          }}
-          noDataPlaceholder={
```

Comment:
we won't show the "empty state component" as table placeholder, but instead of the table